torchmetrics/functional/regression/crps.py

```diff
@@ -0,0 +1,66 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+from torchmetrics.utilities.checks import _check_same_shape
+
+
+def _crps_update(preds: Tensor, target: Tensor) -> Tuple[int, Tensor, float, float]:
+    _check_same_shape(preds[:, 0, ], target)  # second dimension of preds should be number of ensemble members
+
+    batch_size = target.size()[0]
+    n_ensemble_members = preds.size()[1]
```

Comment:
just to be consistent across the codebase
```suggestion
    batch_size = target.shape[0]
    n_ensemble_members = preds.shape[1]
```